src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -124,10 +124,12 @@ export const UserProfileModal: React.FC = () => {
 };
 
 const SuspectedLocation: React.FC<{ user: WithId<User> }> = ({ user }) => {
+  // @debt This will currently load all venues in firebase into memory.. not very efficient
   useFirestoreConnect("venues");
```

Comment:
[note, ?OOS?] `SuspectedLocation` is rendered by `UserProfileModal` only when `ENABLE_SUSPECTED_LOCATION` is set to `true` (it's currently `false`). I believe this is a legacy burn feature.

`UserProfileModal` rendered in `src/pages/VenuePage/TemplateWrapper.tsx`

Reply:
OOS, will handle this in a followup PR.